pkg/cmd/roachtest/testdata/regression.diffs

```diff
@@ -39233,6 +39233,23 @@ diff -U3 --label=/mnt/data1/postgres/src/test/regress/expected/horology.out --la
  SELECT time with time zone '01:30-08' - interval '02:01' AS "23:29:00-08";
   23:29:00-08 
  -------------
+@@ -1034,14 +696,14 @@
+             + interval '02:01' AS time with time zone) AS time) AS "07:31:00";
+  07:31:00 
+ ----------
+- 07:31:00
++ 08:31:00
```

Comment:
What is causing this difference?

Reply:
I wish I knew - I ran the roachtest manually from the gceworker, and I am getting this spurious diffs occasionally. Reverted.